<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogRecherche</class>
 <widget class="QDialog" name="dialogRecherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rechercher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo-bleu.png</normaloff>images/logo-bleu.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidgetRecherche">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane{background:rgb(50,50,50)}
QLineEdit{border:1px solid lightgray;height:25px;}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widgetRechercher">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{background:rgb(50,50,50);color:lightgray;}
QWidget#widgetRechercher{background:rgb(50,50,50);color:lightgray;}
QPushButton{border:1px solid rgb(130,130,130);border-radius:5px;padding:3px 5px;color:rgb(130,130,130);}QPushButton:hover{background:rgb(110,110,110);}QPushButton::enabled{color:lightgray;border-color:lightgray;}</string>
      </property>
      <attribute name="title">
       <string>Rechercher</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-size:13px;</string>
         </property>
         <property name="text">
          <string>Recherchez un mot ou une expression dans le fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="cadreRecherche" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{font-size:13px;}
QWidget#cadreRecherche{border-top:2px solid black;}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Mot ou expression : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="motRechercher">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonRechercher">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Rechercher</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClear">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="resultatRecherche">
              <property name="styleSheet">
               <string notr="true">font-size:13px;border:0;border-right:1px solid lightgray;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Résultat de la recherche ici</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="pushButtonToutSelectionner">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Tout sélectionner</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditProgression">
                 <property name="styleSheet">
                  <string notr="true">border:0;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="pushButtonNext">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Next</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonPrev">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Prev</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetRemplacer">
      <property name="styleSheet">
       <string notr="true">*{background:rgb(50,50,50);color:lightgray;}
QWidget#widgetRechercher{background:rgb(80,80,80);color:lightgray;}
QPushButton{border:1px solid rgb(130,130,130);border-radius:5px;padding:3px 5px;color:rgb(130,130,130);}QPushButton:hover{background:rgb(110,110,110);}QPushButton::enabled{color:lightgray;border-color:lightgray;}</string>
      </property>
      <attribute name="title">
       <string>Remplacer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-size:13px;</string>
         </property>
         <property name="text">
          <string>Remplacer un mot ou une expression dans le fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="cadreRemplacer" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{font-size:13px;}
QWidget#cadreRemplacer{border-top:2px solid black;}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Mot ou expression : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="motRemplacer">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Nouveau :  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nouveauMot">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonRemplacer">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Remplacer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClear2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="infosRemplacement">
            <property name="styleSheet">
             <string notr="true">font-size:13px;;border:0;border-top:1px solid lightgray;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Informations remplacement</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
